lib/worker.browser.js

```diff
@@ -56,14 +75,15 @@ class WebWorker {
             return bridge.emit('uncaughtException', event.error || event.reason);
         }
 
-        if (event.reason) {
+        if (event instanceof PromiseRejectionEvent) {
```

Comment:
note: check if this can cause an issue when the global scope is mutated by bootCode